### PR TITLE
replace location.origin usage with IE11 compatible utils/getLocationOrigin

### PR DESCRIPTION
**Pull request by eezing**
Implements #1120

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -53,3 +53,8 @@ export async function loadGetInitialProps (Component, ctx) {
   }
   return props
 }
+
+export function getLocationOrigin () {
+  const { protocol, hostname, port } = window.location
+  return `${protocol}//${hostname}${port ? ':' + port : ''}`
```
Comment (sedubois) [inline lib/utils.js]: better to use `:${port}` instead of ':' + port

Comment (eezing) [inline lib/utils.js]: Was thinking that too, but the escaped nested template string seemed a little funky. Or am I missing something? 
```javascript
return `${protocol}//${hostname}${port ? /`:${port}/` : ''}`
```

Comment (arunoda) [inline lib/utils.js]: I don't think that's valid. This is just fine.

Comment (sedubois) [inline lib/utils.js]: It's valid, works fine: https://github.com/relatenow/relate/blob/b0181ac9d80e2fee3ab2387e25342c0837bf789f/util/baseUrl.js#L4

```js
`${protocol}//${hostname}${port ? `:${port}` : ''}`
```

Comment (eezing) [inline lib/utils.js]: Yea, you're right. Thanks. Yikes, my snippet is all kinds of broken.

Comment (arunoda) [inline lib/utils.js]: But it feels so wrong and weird. I like the way it is in this PR.

Comment (sedubois) [inline lib/utils.js]: I don't agree, strong concatenation simply shouldn't be used. But anyway it's a detail.

Comment (sedubois) [inline lib/utils.js]: *string

**Pull request merged**